packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.tsx

```diff
@@ -75,6 +77,10 @@ const DesktopDatePicker = React.forwardRef(function DesktopDatePicker<
         hidden: true,
         ...defaultizedProps.slotProps?.toolbar,
       },
+      actionBar: (ownerState: PickerLayoutOwnerState) => ({
+        actions: [],
```

Comment:
Just thought about it, we should be able to memoize the action bar now that it receives no prop and is only using 2 context:

- `usePickerTranslations` which should not change often
- `usePickerActionsContext` which is designed to never change

This empty array would break the memoization, so maybe moving it outside of the render would be better.
Same for all the other occurences where we define the actions in the render like this.

Reply:
Great observation! 👍 
But based on my exploration, I don't think `PickersActionBar` can ever be memoizable if we keep using `useSlotProps` and supporting `slotProps` callback. 🤔 
Not sure if it's worth changing anything now until we decide on what to do with this blocker. 🤔